application/F3DStarter.cxx

```diff
@@ -843,23 +843,46 @@ int F3DStarter::Start(int argc, char** argv)
         return true;
       });
 
-    interactor.addInteractionCommand(
-      "Left", f3d::interactor::ModifierKeys::NONE, "load_previous_file_group");
-    interactor.addInteractionCommand(
-      "Right", f3d::interactor::ModifierKeys::NONE, "load_next_file_group");
-    interactor.addInteractionCommand(
-      "Up", f3d::interactor::ModifierKeys::NONE, "reload_current_file_group");
-    interactor.addInteractionCommand(
-      "Down", f3d::interactor::ModifierKeys::NONE, "add_current_directories");
-    interactor.addInteractionCommand(
-      "F11", f3d::interactor::ModifierKeys::NONE, "take_minimal_screenshot");
-    interactor.addInteractionCommand("F12", f3d::interactor::ModifierKeys::NONE, "take_screenshot");
+    // This replace an existing command in libf3d
+    interactor.addCommandCallback("add_files",
```

Comment:
I'm wondering if we shouldn't have a way to specify we are replacing an existing command.
This way, the user would be able to catch a change in libf3d command name for example.  
Do you think we could add `replaceCommandCallback` and maybe throw if calling `addCommandCallback` is called with an existing command name?

Reply:
I'm not against it, but it does make the API a bit heavier. I do plan to add a `addDefaultCommandCallbacks` and a `removeAllCommandCallbacks` in a dedicated PR, maybe I should add your suggestion in the same PR ?

Reply:
fair enough